metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -1174,6 +1413,22 @@ def emit_datasource(
             for wu in workunits:
                 self.report.report_workunit(wu)
                 yield wu
+        elif (
+            datasource.get("luid")
```

Comment:
Published Data Source -> why the container construction is different for embedded vs published data sources?

def extract_container_for_published_data_source()

Reply:
I have added comments. for embedded datasources the container is workbook , whereas for published datasources the parent container is project